packages/server/api/controllers/favorites.controller.js

```diff
@@ -0,0 +1,35 @@
+const knex = require('../../config/db');
+const HttpError = require('../lib/utils/http-error');
+
+const getFavorites = async () => {
+  try {
```

Comment:
Why is there a try/catch in this handler but not in the other 2? Is this action more risky than the others?
Should you remove this try/catch or add error handling to the other 2?

Reply:
Yes I will update this

Reply:
I have updated try /catch in other two